src/cli/index.js

```diff
@@ -98,6 +100,7 @@ export function main({
     'prepend the node executable dir to the PATH in scripts',
     boolify,
   );
+  commander.option('--no-node-version-check', 'do not warn when using a potentially unsupported Node version');
```

Comment:
That's a long command name :D

Reply:
`--no-node-version-check-or-else-youll-be-fired`